README.md

```diff
@@ -124,6 +125,12 @@ repository.
 
 For more detailed usage of the package functionality, refer: [https://idealo.github.io/imagededup/](https://idealo.github.io/imagededup/)
 
+## Benchmarks
```

Comment:
would be nice to have some kind of logo for Benchmarks as well, looks weird in the rendered markdown otherwise

![image](https://user-images.githubusercontent.com/19587525/71616129-fc95ba00-2bb4-11ea-803e-2d15fa9963fc.png)

Reply:
maybe ⏳?

Reply:
Yeh I saw that as well and added a logo already. I took a lab tube instead of hour glass but both look good. @tanujjain should decide at the end.

Reply:
I like ⏳ better, will make the change.